spec/models/classification_spec.rb

```diff
@@ -571,6 +571,31 @@
     end
   end
 
+  describe '#save' do
+    let(:new_name) { "new_tag_name" }
+
+    context "editing existing tag" do
+      it "updates record in Tag table which linked to this classification" do
+        classification = FactoryBot.create(:classification_tag, :name => "some_tag_name")
+        tag = classification.tag
```

Comment:
think the failure is when you create a new tag here.

I know I suggested getting away from assigning a tag here.
And we should not be assigning a tag here.
But I think the error is because we assign a tag here.
So... We can either:
1. Remove all the cases that we assign a tag here (because the code is bogus)
2. Change the model to work with that bogus case (**<== I'm suggesting this path**)

Reply:
👍 to 2.